.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,6 @@
+name: "Custom CodeQL Config"
```

Comment:
Can we give this a more descriptive name?

Reply:
Done